### PR TITLE
Updates to three topics to add info for account aliases.

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Updated three topics to add information about account aliases.

## Changes

Three topics were updated. Added alias to glossary. Added info about aliases to concordium client and to managing accounts topics.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

**Comment by dg-concordium** [comment]
On hold until around 17 Jan. 2022 as per: https://concordium.atlassian.net/wiki/spaces/GEN/pages/908296207/Release+R2-1

File: source/shared/net/references/concordium-client.rst [hunk 1]
```diff
@@ -284,6 +284,19 @@ The ``--name`` option selects which account to import and imports it with this
 name. If it's omitted, all accounts in the file are imported under their
 existing names.
 
+Generate account aliases
```
Comment (jens-concordium) [inline source/shared/net/references/concordium-client.rst:287]: Maybe this should called "View" or "Show" account aliases, as I guess they exist already. As the command is called `show-alias`, the title "Show account aliases" might be right

File: source/shared/net/references/manage-accounts.rst [hunk 2]
```diff
@@ -108,6 +108,28 @@ The Mobile Wallet keeps track of the sequence number and assigns the correct one
...
+
+In protocol versions 1 and 2 accounts and account addresses have a one-to-one relationship. In protocol version 3 each account has 16777216 addresses, namely a so-called canonical account address together with
+matching account aliases. The canonical account address is derived when an account is created on chain. The other 16 million addresses with matching initial 29 bytes are referred to as account aliases for
+the same account. Thus, accounts can be referred to by any address whose initial 29 bytes match.
+
+This allows each account to have aliases for different uses and creates a kind of sub-account structure. An account owner can give out different aliases for different uses to keep track of transfers and assign them meaning.
+
+Each account still has one total account balance. Hence, transfers to and from aliases of an account add to and subtract from that total account balance, respectively. Transfers between different aliases of the same account do not change the balance of the account, apart from cost. Finalization, block, and baking rewards are always received on the account's canonical address.
+
+To generate aliases, enter:
```
Comment (jens-concordium) [inline source/shared/net/references/manage-accounts.rst:122]:
```suggestion
To show aliases, enter:
```

**Review by abizjak — APPROVED** [review]
I agree with Jens' suggestions, but otherwise good.

**Pull request merged**